scripts/dats_validator/examples/valid_dats.json

```diff
@@ -68,7 +68,7 @@
   "distributions": [
     {
       "formats": [
-        ".VCF",
+        "VCF",
```

Comment:
@cmadjar You have to change ".VCF" to "VCF" everywhere (subdatasets in hasPart)  in the file to make the file valid.